cake/build-settings.cake

```diff
@@ -1,23 +1,354 @@
-#load ./ci.cake
-#load ./packaging.cake
-#load ./package-checks.cake
-#load ./test-results.cake
-#load ./package-tests.cake
-#load ./package-tester.cake
-#load ./header-check.cake
-#load ./local-tasks.cake
-
-public class BuildSettings
+public static class BuildSettings
 {
-    public BuildSettings(ISetupContext context)
-    {
+	private static BuildSystem _buildSystem;
+
+	public static void Initialize(
+      // Required parameters
```

Comment:
Odd whitespace?

Reply:
May be tabs vs spaces here

Reply:
Actually this line and the next both use spaces and are still different (but the `public static void Initialize` uses tabs).  So fun all over the place. :D

Reply:
Problem is that I keep experimenting with different approaches to editing cake files. Also, I've copied some from elsewhere. I'll clean up as much as I can and try to settle on one extension for editing them.